### PR TITLE
(imp) [SD-3650] Page refreshing is required for vocabulary changes to take effect

### PR DESCRIPTION
**Pull request by sivakuna-aap**
Resolves: [SD-3650](https://dev.sourcefabric.org/browse/SD-3650)
Requires: https://github.com/superdesk/superdesk-core/pull/81 and https://github.com/superdesk/superdesk/pull/1391

### CODE REVIEW

**Comment by plamut** [comment]
:+1:

**Comment by akintolga** [comment]
:+1:

**Comment by mdhaman** [comment]
:+1:

File: server/features/vocabularies.feature [hunk 1]
```diff
@@ -82,3 +82,19 @@ Feature: Vocabularies
...
+
+  @auth @notification
+  Scenario: User receives notification when a vocabulary is updated
+    Given the "vocabularies"
+    """
+    [{"_id": "categories", "display_name": "Category", "items": [{"name": "National", "value": "A", "is_active": true}, {"name": "Domestic Sports", "value": "T", "is_active": false}]}]
+    """
+    When we patch "/vocabularies/categories"
+    """
+    {"_id": "categories", "display_name": "Category", "items": [{"name": "National", "value": "A", "is_active": true}, {"name": "Domestic Sports", "value": "T", "is_active": true}]}
```
Comment (mdhaman) [inline server/features/vocabularies.feature]: Behave test are failing at this step. Remove `_id` from the payload.

**Pull request merged**